OpenRA.Mods.Common/UtilityCommands/ExtractTraitDocsCommand.cs

```diff
@@ -75,10 +75,40 @@ public void Run(ModData modData, string[] args)
 					}
 				}
 
+				var forbiddenBaseTypes = new[]
+				{
+					typeof(ITraitInfo),
+					typeof(TraitInfo<>),
+				};
+
+				var upgradableIndicator = "OpenRA.Mods.Common.Traits.UpgradableTraitInfo";
+				var upgradable = HasSuperType(t, upgradableIndicator);
+				doc.AppendFormat("Upgradable: {0}", upgradable ? "Yes" : "No");
```

Comment:
I opted for a clear yes/no over just 'upgradable' so you could see it on all traits.